openapi.yaml

```diff
@@ -527,6 +550,11 @@ components:
             additionalProperties:
               type: integer
               format: int32
+    JSONObject:
```

Comment:
Can we name this something relevant like `SystemDetails`? Even if nothing is implemented yet?

Reply:
I simply implemented it as [T129272616](https://www.internalfb.com/intern/tasks/?t=129272616) says. But I am ok with both way. Maybe I could do it separately. what's your opinion @elludraon ?

Reply:
Oh, I see, it's fine as is then.